build/media-suite_compile.sh

```diff
@@ -2070,6 +2070,7 @@ if [[ $ffmpeg != no ]] && enabled libxeve &&
     mv -f "$LOCALDESTDIR"/lib/xeve/libxeve.a "$LOCALDESTDIR"/lib/libxeve.a
     mv -f "$LOCALDESTDIR"/lib/libxeve.dll.a "$LOCALDESTDIR"/lib/libxeve.dll.a.dyn
     do_checkIfExist
+    rmdir "$LOCALDESTDIR/lib/xeve" > /dev/null 2>&1
```

Comment:
Can this have a note about why this is explicitly done here? Also, can you move this right next to the `mv` to make it clear that the reason this dir is not needed is that the stuff inside has been moved out?

Reply:
What note? Like a comment? Shall it be about `rmdir` only or including why do `mv` as well?

Reply:
Yeah, a comment as why this is fine, something like `# delete empty folder`. Just so later on, if we see this, we know that we are assuming the folder is empty due to the previous `mv`s.